app/controllers/redhat_access/machine_telemetries_controller.rb

```diff
@@ -0,0 +1,71 @@
+module RedhatAccess
+  class MachineTelemetriesController < ::Api::V2::BaseController
+    layout false
+
+    include ::RedhatAccess::ClientAuthentication
+    include ::RedhatAccess::CandlepinRelated
+
+    before_action :ensure_telemetry_enabled_for_consumer, :only => [:forward_request]
```

Comment:
I suppose it's not needed - if insights-client decides it want to send data, it means the user has enabled it. I suppose we can pass it to the cloud, and make it cloud's responsibility to reject requests without proper subscriptions.

Reply:
The reason we had this in the previous plugin was to give the Satellite admin control over whether the proxy functionality should be allowed.   This useful when the Satellite owner is not the same as the managed host owner. I suggest we keep this as its used in practice.